Jenkins-dev-Frontiers_317_delete_products.py

```diff
@@ -0,0 +1,82 @@
+"""
+Working on a script to clean up demo store. delete all products without images.
+Example command to run script: python Jenkins-dev-Frontiers_317_delete_products.py --confirm-delete yes
+"""
+
+import argparse
+import logging
+import os
+from woocommerce import API
+
+# defining argument parser
+parser = argparse.ArgumentParser(description="Delete products with no images from the database")
+parser.add_argument("--confirm-delete", choices=["yes", "no"], default="no", help="Confirm deletion (yes/no)")
```

Comment:
may be in the help explain what happens if 'no'. It will jest report how many would have been deleted but not actually delete it. It is like saying 'run in report mode'.